test/e2e/e2e_test.go

```diff
@@ -28,6 +29,7 @@ const (
 	retryInterval        = 1 * time.Second  // polling interval timer
 	retryTimeout         = 40 * time.Second // polling timeout
 	ciNetworkName        = "kind"
+	agnhostImage         = "k8s.gcr.io/e2e-test-images/agnhost:2.21"
```

Comment:
2.26 bring some fixes for iperf, that are needed for our multicast e2e tests
```suggestion
	agnhostImage         = "k8s.gcr.io/e2e-test-images/agnhost:2.26"
```